packages/itwin/property-grid/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@itwin/property-grid-react",
-  "version": "1.12.0",
+  "version": "1.13.0",
```

Comment:
Is this intentional?

Reply:
Because I deprecated `PropertyGridUiItemsProvider` the package version should be changed.

Reply:
changed back